package.json

```diff
@@ -68,12 +68,14 @@
     "nunjucks": "^3.2.4",
     "pino": "^9.4.0",
     "pino-pretty": "^11.2.2",
-    "undici": "^6.19.8"
+    "undici": "^6.19.8",
+    "util": "0.12.5"
```

Comment:
I don't see us using this dependency in the code below - presumably this was a dependency for frontend tests that we've since removed?

Reply:
no - this was a audit fix thing

Reply:
but that mightve been for journey tests - ill test to make sure

Reply:
oh you mean util - yes thats not needed anymore - the comments made it look like it was undici